syncd/Syncd.cpp

```diff
@@ -920,6 +1097,35 @@ sai_status_t Syncd::processBulkOid(
 
     sai_status_t all = SAI_STATUS_SUCCESS;
 
+#ifdef SAI_SUPPORT_BULK_API
+
+    sai_bulk_op_error_mode_t mode = SAI_BULK_OP_ERROR_MODE_IGNORE_ERROR;
+
+    switch (api)
+    {
+        case SAI_COMMON_API_BULK_CREATE:
+            all = processBulkOidCreate(objectType, mode, objectIds, attributes, statuses);
+            break;
+
+        case SAI_COMMON_API_BULK_REMOVE:
+            all = processBulkOidRemove(objectType, mode, objectIds, statuses);
+            break;
+
+        default:
+            all = SAI_STATUS_NOT_SUPPORTED;
+            SWSS_LOG_THROW("api %d is not supported in bulk mode", api);
+    }
+
+    if (all != SAI_STATUS_NOT_SUPPORTED && all != SAI_STATUS_NOT_IMPLEMENTED)
```

Comment:
remove this check, since in syncd mode we also required to send response to sender, and in line 1116 you throw anyway

Reply:
the same as above, in 856 line